### PR TITLE
Technical Documentation

### LINKED ISSUE #54: NFR1 provide technical and design documentation

### PR DESCRIPTION
**Pull request by sonlinux**
fix #54 
Moved PR from django-bims to here since this is where we have custom apps we may need to add to the documentation.

### CODE REVIEW

**Comment by sonlinux** [comment]
- [ ] Added readme on how to generate documentation
- [ ] Generated documentation for an example apps model (bims)

Example model documentation

sample_documentation

File: django_project/docs/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+``` python manage.py listing_models --app fish
+```
+--output(-o)
+It writes the results to the specified file.
+
+```python manage.py listing_models --output index.rst
+```
+--format(-f)
+You can choice output format. rst (reStructuredText) or md (Markdown). Default format is rst.
```
Comment (timlinux) [inline django_project/docs/README.md]: choice -> choose

File: django_project/docs/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+==============================
+
+## Technical documentation for LEDET-BIMS Project.
+
+## Firstly you need to be in the base project directory where the management
+ command is run from.
+
+```docs/docs/```
+
+**Then to generated the documentation run **
```
Comment (timlinux) [inline django_project/docs/README.md]: generated -> generate

**Comment by timlinux** [comment]
Looking nice @sonlinux 

Please move to kartoza-bims and then just register any ledet specific models when we add them to also have documentation generated.

The README must be referenced from the top level README so that it is discoverable.

The readme should also include notes on how to generate the spinx docs to html and I would like to see a copy  of the docs rendered to a web site somewhere - ideally with a single command that does everything.

**Pull request closed without merging**